Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -276,13 +282,54 @@ private void OnGUI()
             try
             {
                 RenderInteractionList(currentInteractionList, currentControllerMapping.HasCustomInteractionMappings);
+                RenderMappingList(mappedControllerList);
             }
             catch (Exception)
             {
                 thisWindow.Close();
             }
         }
 
+        private void RenderMappingList(List<string> controllerList)
```

Comment:
Should this method null check `controllerList` before it's used? Since the only setter I can find is in `Show`, there might be a chance something calls that method without passing a valid list, so it could still be null. (this might be another scenario like my other comment that can't be hit by any current code, but it still might be worth protecting against)

That, or just initialize `mappedControllerList` to an empty list on line 83 I think, since the null check in `Show` will prevent it from ever being set to null.